docs/prometheus.md

```diff
@@ -9,9 +9,6 @@ avoid exposing the entire admin interface to Prometheus (and other workloads in
 the cluster), Contour configures a static listener that sends traffic to the
 stats endpoint and nowhere else.
 
-To enable the static listener, set the `--statsd-enabled` flag on the Contour
```

Comment:
Should we add docs here for --stats-address and --stats-port?

Reply:
Yup good point @rosskukulinski, I think we missed those docs when this was introduced initally.